pkg/config/config.go

```diff
@@ -247,6 +248,12 @@ type LintersSettings struct {
 	Custom map[string]CustomLinterSettings
 }
 
+type GoHeaderSettings struct {
```

Comment:
Can we have some example for settings in https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml ? 

This will help for quick reference, compared to checking the original repo or actual implementation.

Reply:
>Can we have some example for settings in https://github.com/golangci/golangci-lint/blob/master/.golangci.example.yml ?

Sure, fixed